.github/workflows/docker-build.yaml

```diff
@@ -16,67 +16,99 @@ jobs:
     build-docker-images:
         if: github.event.pull_request.head.repo.full_name == 'shopsys/shopsys'
         name: Build PHP-FPM and Elasticsearch images
-        needs: cancel
         runs-on: ubuntu-20.04
+        permissions:
+            contents: read
+            packages: write
         steps:
             -   name: GIT checkout branch - ${{ github.ref }}
-                uses: actions/checkout@v2
+                uses: actions/checkout@v3
                 with:
                     ref: ${{ github.ref }}
-            -   name: Login to Docker Hub
-                uses: docker/login-action@v1
+            -   name: Login to GitHub Container Registry
+                uses: docker/login-action@v2
                 with:
-                    username: ${{ secrets.DOCKER_USERNAME }}
-                    password: ${{ secrets.DOCKER_PASSWORD }}
+                    registry: ghcr.io
+                    username: ${{ github.actor }}
+                    password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Does it need some settings for other team members? 
The actor is the one who pushed, right (I see TomasLudvik in the output), so when somebody else with the maintainer permission creates a PR, will it be accepted with the token?

Reply:
Yes, permissions says that GITHUB_TOKEN has to be created with write permissions for packages, so nothing else is required for other maintainers.